pkg/proxy/snapshot.go

```diff
@@ -43,7 +44,30 @@ func (p *Proxy) SnapshotList(ctx context.Context, req *rpc.ProxyEngineRequest) (
 		Disks: map[string]*rpc.EngineSnapshotDiskInfo{},
 	}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.